waiter/src/waiter/token.clj

```diff
@@ -74,6 +79,16 @@
    :last-update-time last-update-time
    :maintenance (some? maintenance)})
 
+(defn make-index-event
+  "Create an event for watch endpoints"
+  [type object]
+  {:object object :type type})
+
+(defn send-internal-index-event
+  "Send an internal event to be processed by the tokens-watch-maintainer daemon process"
+  [tokens-update-chan token owner]
+  (async/put! tokens-update-chan {:owner owner :token token}))
```

Comment:
Let's add a log here, so we can track which events get picked up from the logs while debugging.